tools/llvm-kompile-compute-loc/main.cpp

```diff
@@ -0,0 +1,102 @@
+#include <kllvm/ast/AST.h>
+#include <kllvm/parser/KOREParser.h>
+
+#include <llvm/Support/CommandLine.h>
+
+#include <cstdlib>
+#include <fstream>
+#include <iostream>
+#include <string>
+
+using namespace llvm;
+using namespace kllvm;
+
+cl::OptionCategory loc_cat("llvm-kompile-compute-loc options");
+
+cl::opt<std::string> kompiled_dir(
+    cl::Positional, cl::desc("<kompiled-dir>"), cl::Required, cl::cat(loc_cat));
+
+cl::opt<std::string> ordinal(
+    cl::Positional, cl::desc("<ordinal>"), cl::Required, cl::cat(loc_cat));
```

Comment:
Ordinals are integers; you can avoid a parsing step here by declaring a `cl::opt<int>` directly